.gitignore

```diff
@@ -8,7 +8,6 @@ _obj
 _test
 .cover/
 .idea/
-.vscode/
```

Comment:
Can we add this back in? I think it would be best if we try to be unopinionated on this subject.

Reply:
I forgot to undo this when I removed the .vscode files from the PR